python/ray/actor.py

```diff
@@ -265,6 +267,21 @@ def _bind(
         }
         actor._ray_dag_bind_index += 1
 
+        # Run it to verify if args match signature.
+        try:
+            signature.flatten_args(self._signature, args, kwargs)
+        except TypeError as e:
+            signature_copy = self._signature.copy()
+            if len(signature_copy) > 0 and signature_copy[-1].name == "_ray_trace_ctx":
+                # Remove the trace context arg for readability.
+                signature_copy.pop(-1)
+            signature_copy = inspect.Signature(parameters=signature_copy)
+            raise TypeError(
+                f"{str(e)}. The function `{self._method_name}` has a signature "
+                f"`{signature_copy}`, but it doesn't match a given argument to a "
+                f"bind function. args: {args}. kwargs: {kwargs}."
+            ) from None
```

Comment:
Should this be wrapped into another util method like `flatten_args` in `signature.py`?

Reply:
I created a new API validate_args.

For this question, I feel like adding error message here is better (since it needs to access several private attribute. Feel awkward to create a separate util func)